app/src/main/java/ua/kozlov/madfood/Foods.java

```diff
@@ -0,0 +1,265 @@
+package ua.kozlov.madfood;
+
+
+import android.content.Context;
+
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Foods implements FoodsInterface {
+    @Override
+    public ArrayList<String> getFoodsList(String groupName, Context context) {
+        XmlPullParser parser = getParser(groupName, context);
+        ArrayList<String> foodsList = setFoodNamesToList(parser);
+        return foodsList;
+    }
+
+    @Override
+    public String getFoodName(String groupID, String id, Context context) {
+        return null;
+    }
+
+    @Override
+    public String getFoodId(String groupID, String name, Context context) {
+        return null;
+    }
+
+    @Override
+    public ArrayList<String> getFoodParameters(String groupName, int position, Context context) {
```

Comment:
@OlehKozlov if **position** does't be changed in method, it must be **const**
Please, check all parametrs in all methods